les/api_backend.go

```diff
@@ -91,6 +91,10 @@ func (b *LesApiBackend) SetHead(number uint64) {
 	b.eth.blockchain.SetHead(number)
 }
 
+func (b *LesApiBackend) SetL1Head(number uint64) {
+	panic("unimplemented")
```

Comment:
Is there any way to trigger this panic at all that we haven't accounted for?

Reply:
Nope, the light client server isn't used at all